lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/EvidenceFactory.java

```diff
@@ -35,15 +37,29 @@ public class EvidenceFactory {
     private static final int UNSUITABLE_QUESTION_QUALITY = 0;
     private final EventProbe eventProbe;
     private final ObjectMapper objectMapper;
+    private String kbvQuestionStrategy;
     private final Map<String, Integer> kbvQualityMapping;
 
+    void setKbvQuestionStrategy(String questionStrategy) {
+        this.kbvQuestionStrategy = questionStrategy;
+    }
+
+    public String getKbvQuestionStrategy() {
+        return this.kbvQuestionStrategy;
+    }
+
+    public Map<String, Integer> getKbvQualityMapping() {
+        return kbvQualityMapping;
+    }
+
     public EvidenceFactory(
             ObjectMapper objectMapper,
             EventProbe eventProbe,
             Map<String, Integer> kbvQualityMapping) {
         this.objectMapper = objectMapper;
         this.eventProbe = eventProbe;
         this.kbvQualityMapping = kbvQualityMapping;
+        this.kbvQuestionStrategy = "3 out of 4 Prioritised";
```

Comment:
I don't think so, getting the value from configuration would cause more changes, detracting from the purpose of this PR. We have make changes in the VerifiableCredentialService [here](https://github.com/govuk-one-login/ipv-cri-kbv-api/blob/main/lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/VerifiableCredentialService.java#L63) kbvQualityMapping mapping also comes from configurationService

We might want to pass in a single object that has both configuration   
`
this.evidenceFactory =
                new EvidenceFactory(this.objectMapper, new EventProbe(), **kbvQualityMapping**)`

Also, I didn't want any other changes that are not strictly related to CheckDetails generation. keeping the EvidenceFactory as close to what it is here is beneficial for review. And also for the next PR on renaming stuff and comparing with what was there before

Reply:
Happy with that thanks Ola